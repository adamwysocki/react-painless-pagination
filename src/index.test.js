// index.test.js
import React from "react";
import PainlessPagination from "./index";
import renderer from "react-test-renderer";
import Enzyme, { shallow, render, mount } from "enzyme";
import sinon, { stub } from "sinon";
import Adapter from "enzyme-adapter-react-16";

// React 16 Enzyme Adapter
Enzyme.configure({ adapter: new Adapter() });

it("should render", () => {
  const onStateSelect = stub();
  const component = renderer.create(<PainlessPagination />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
