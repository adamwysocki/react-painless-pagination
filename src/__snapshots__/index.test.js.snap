// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="sc-bdVaJa khUSez"
>
  <ul
    className="sc-bwzfXH bPeWBT"
  >
    <a
      className="sc-htpNat jZGOfu"
      href="/races/1"
    >
      <li
        className="sc-ifAKCX dcZKHI"
      >
        <i />
      </li>
    </a>
    <a
      className="sc-htpNat jZGOfu"
      href="/races/2"
    >
      <li
        className="sc-EHOje ePrfLX"
      >
        <i />
      </li>
    </a>
  </ul>
</div>
`;
